### DISCUSSION TITLE
Is there any plan to update scoped-elements to support lit 2?

### DISCUSSION THREAD

**Discussion by nbamfordcirium** [open-wc/open-wc#2674] [Q&A] [answered]
I'm attempting to use `@lit-labs/nextjs` to enable server side rendering in next for my web components, but `scoped-elements` has this dependency: ` "@lit/reactive-element": "^1.0.0"`, which is incompatible with this solution.

Is supporting lit v2 on the roadmap anywhere?

**Comment by Westbrook** [comment, marked as answer]
`@lit/reactive-element`'s latest version is `1.6.1` which is resolvable from `^1.0.0` see https://semver.npmjs.com/ and leverage those values, if you need to be sure. In that way, `scoped-elements` already supports `lit@2.0`...

**Reply by nbamfordcirium** [reply]
Thank you!
